.github/workflows/generate-desktop-targets.yml

```diff
@@ -78,11 +78,21 @@ jobs:
           FLEET_DESKTOP_VERSION=${{ needs.set-version.outputs.FLEET_DESKTOP_VERSION }} \
           make desktop-app-tar-gz
 
+      - name: Extract archive for attestation
+        run: |
+          mkdir -p extracted
+          tar -xzf desktop.app.tar.gz -C extracted
+
       - name: Attest binary
         continue-on-error: true
         uses: actions/attest-build-provenance@619dbb2e03e0189af0c55118e7d3c5e129e99726 # v2.0
         with:
-          subject-path: "desktop.app.tar.gz"
+          subject-path: "extracted/*.pkg"
```

Comment:
For this I think we want to attest `/Contents/MacOS/fleet-desktop_amd64` and `/Contents/MacOS/fleet-desktop_arm64` (based on what I'm seeing [here](https://github.com/fleetdm/fleet/blob/main/tools/desktop/desktop.go#L137-L176))